hcl/hcl.go

```diff
@@ -214,7 +254,11 @@ func Parse(fname string, data []byte) (Config, error) {
 		)
 	}
 
-	var tmconfig Config
+	tmconfig, err := NewConfig(filepath.Dir(fname), "")
+	if err != nil {
+		return Config{}, fmt.Errorf("parsing HCL file %q: %w", fname, err)
```

Comment:
```suggestion
		return Config{}, fmt.Errorf("creating config %q: %w", fname, err)
```

I think NewConfig doesn't to any parsing, but could have missed something :thinking: